specs/chains/chains.spec

```diff
@@ -0,0 +1,40 @@
+PIPELINES-27
+# Tekton Chains tests
+
+Precondition:
+* Validate Operator should be installed
+
+## Using Tekton Chains to create and verify task run signatures: PIPELINES-27-TC01
+Tags: chains, e2e, taskrun, sanity
+Component: Chains
+Level: Integration
+Type: Functional
+Importance: Critical
+
+Steps: 
+    * Update the TektonConfig with taskrun format as "in-toto" taskrun storage as "tekton" oci storage as "" transparency mode as "false"
+    * Apply
+     | S.NO | resource_dir                          |
+     |------|---------------------------------------|
+     | 1    | testdata/chains/task-output-image.yaml|
+    * Verify "taskrun" signature
```

Comment:
This step fails because the file `sign` doesn't exist
```
* Verify "taskrun" signature       ...[FAIL]
2024/03/22 15:43:46 Skipping deletion of the namespace 'releasetest-wcr78' as the test got failed
        
        Failed Step: Verify "taskrun" signature
        Specification: specs/chains/chains.spec:20
        Error Message: [assertion failed: 
        Command:  cosign verify-blob-attestation --insecure-ignore-tlog --key /home/ppitonak/workspace/release-tests/testdata/chains/key/cosign.pub --signature sign --type slsaprovenance --check-claims=false /dev/null
        ExitCode: 1
        Error:    exit status 1
        Stdout:   
        Stderr:   WARNING: Skipping tlog verification is an insecure practice that lacks of transparency and auditability verification for the blob attestation.
        Error: getting ctlog public keys: updating local metadata and targets: error updating to TUF remote mirror: invalid key
```

Reply:
You can do `file, err := os.Create(resource.Path("sign"))` and then `"--signature", file.Name()` in the command